### PR TITLE
Refs #5388 Fixed whitelist and bug with xml structure literal include. [5389]

### PR DESCRIPTION
**Pull request by LuisGP**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/88/)

File: code/XMLTester.xml [hunk 1]
```diff
@@ -138,9 +138,9 @@
 <publisher profile_name="publisher_xml_conf_disable_positive_acks_profile">
     <qos>
         <disablePositiveAcks>
-	    <enabled>true</enabled>
+        <enabled>true</enabled>
```
Comment (MiguelCompany) [inline code/XMLTester.xml]:
```suggestion
            <enabled>true</enabled>
```

File: code/XMLTester.xml [hunk 2]
```diff
@@ -1569,12 +1569,12 @@
             <publishMode>
                 <kind>ASYNCHRONOUS</kind>
             </publishMode>
-	    <disablePositiveAcks>
-	        <enabled>true</enabled>
-	        <duration>
-		    <sec>1</sec>
-		</duration>
-	    </disablePositiveAcks>
+        <disablePositiveAcks>
```
Comment (MiguelCompany) [inline code/XMLTester.xml]: Indentation on this block

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/90/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/91/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/92/)

**Pull request merged**